src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -129,4 +132,25 @@
         @Config.Comment("Array of blacklisted dimension IDs in which Air Collector does not work.")
         public int[] airCollectorDimensionBlacklist = new int[]{};
     }
+
+    public static class NanoSaberConfiguration {
+
+        @Config.RangeDouble(min = 0, max = 100)
+        @Config.Comment("The additional damage added when the NanoSaber is powered. Default: 20f")
```

Comment:
I think that players should be able to copy and paste default value to config and it should work so this value should be 20.0

Reply:
This has been done.